metagraph/tests/algorithms/__init__.py

```diff
@@ -48,6 +48,19 @@ def __init__(
 
         self.plans = plans
 
+    def _translate_atomic_type(self, value, dst_type, algo_path):
+        try:
+            if dst_type in (float, int):
```

Comment:
Rather than excluding these types specifically, we should check if dst_type is a ConcreteType (or converts to a one using `resolver.class_to_concrete`). Only translate for concrete types. Ignore others.

This would also allow line 90 to be a simple `else:` because the `None` check will be caught here.

Reply:
https://github.com/metagraph-dev/metagraph/pull/117/commits/8c2d7ac49f32a6605b5b02a8ed9758c2591165bf implements these changes.

Reply:
@jim22k , https://github.com/metagraph-dev/metagraph/pull/117/commits/8351d986d41c2cafc9c57d25e645cd0b88a956df puts back in the `elif expected_type is not None` line since it's still needed to handle the default value for the `expected_type` parameter of `MultiVerify.custom_compare`.